packages/atlas/src/components/ActionBar/ActionBar.tsx

```diff
@@ -40,6 +42,7 @@ export type ActionBarProps = {
 export const ActionBar = forwardRef<HTMLDivElement, ActionBarProps>(
   ({ fee, feeLoading, isActive = true, className, primaryButton, secondaryButton, infoBadge }, ref) => {
     const smMatch = useMediaMatch('sm')
+    const { actionHandler, loadingState } = useWaitForFee(!!feeLoading, fee, primaryButton.onClick)
```

Comment:
```suggestion
    const { actionHandler, loadingState } = useWaitForFee(feeLoading, fee, primaryButton.onClick)
```
I assume `feeLoading` is a boolean

Reply:
But it may be undefined here